rotkehlchen/tests/api/test_history.py

```diff
@@ -208,7 +208,7 @@ def test_query_history_remote_errors(rotkehlchen_api_server_with_exchanges: 'API
     warnings = rotki.msg_aggregator.consume_warnings()
     assert len(warnings) == 0
     errors = rotki.msg_aggregator.consume_errors()
-    assert len(errors) == 2
+    assert len(errors) == 3
```

Comment:
why this change?

Reply:
The error being checked here is this:
```
Failed to query kraken ledger between 0 and 1733842067. Kraken remote error
```
Which happens in `query_kraken_ledgers` which is called in these functions:
* `query_online_trade_history`
* `query_online_deposits_withdrawals`
* `query_history_events`

There were two of these errors originally because the `query_history_with_callbacks` function was being used, which queried both trades and deposit/withdrawals. But now `query_history_events` has also been added to `query_history_with_callbacks` and now this error appears three times, due to all three of the above functions being called.

After the kraken exchange interface is updated to use the new AssetMovements, `query_online_deposits_withdrawals` will be merged with `query_history_events` and this should go back to only two errors here.

Reply:
okay can you add a comment then explaining that this will be reverted soon and why. And delete it when making it go back to two errors.

Reply:
added a comment